.env.example

```diff
@@ -128,3 +128,5 @@ INSIGHTLY_API_KEY=
 
 SENTRY_LARAVEL_ENABLED=true
 SENTRY_LARAVEL_DSN=
+
+GTAG_MEASUREMENT_ID=
```

Comment:
I think we need to add this to appconfig repo also https://github.com/cultuurnet/appconfig/tree/main/files/platform